python/artm/regularizers.py

```diff
@@ -588,3 +589,36 @@ def __init__(self, name=None, tau=1.0, topic_names=None,
                                       config=config,
                                       topic_names=topic_names,
                                       alpha_iter=alpha_iter)
+
+
+class BitermsPhiRegularizer(BaseRegularizerPhi):
```

Comment:
At the top of ``regularizers.py`` file there is a variable called ``__ALL__``. Either we should remove it (if its not used), or add there new regularizer.